Marlin/src/pins/stm32f1/pins_BTT_SKR_E3_DIP.h

```diff
@@ -21,7 +21,7 @@
  */
 #pragma once
 
-#if NOT_TARGET(TARGET_STM32F1)
+#if NOT_TARGET(__STM32F1__, STM32F1)
   #error "Oops! Select an STM32F1 board in 'Tools > Board.'"
 #endif
 
```

Comment:
You might want to use #include "env_validate.h" instead

Reply:
Thanks!